### PR TITLE
[Swift 5] fix Multipart FormData encoding and add support for Form URLEncoded enconding

### PR DESCRIPTION
**Pull request by 4brunu**
Previously it was only possible to use the encoding `multipart/form-data` when sending files.
With this PR that is fixed.

And it also adds support for 'application/x-www-form-urlencoded' fixing https://github.com/OpenAPITools/openapi-generator/issues/8169

This PR also contains some fixes on code formatting.

### PR checklist
 
- [X] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [X] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [X] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
- [X] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [X] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `5.1.x`, `6.0.x`
- [X] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

@jgavris (2017/07) @ehyche (2017/08) @Edubits (2017/09) @jaz-ah (2017/09) @4brunu (2019/11)

### CODE REVIEW

File: modules/openapi-generator/src/main/resources/swift5/APIs.mustache [hunk 1]
```diff
@@ -28,13 +27,16 @@ import Foundation
...
-    required {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} init(method: String, URLString: String, parameters: [String:Any]?, isBody: Bool, headers: [String:String] = [:]) {
+    required {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} init(method: String, URLString: String, parameters: [String:Any]?, headers: [String:String] = [:]) {
         self.method = method
         self.URLString = URLString
         self.parameters = parameters
-        self.isBody = isBody
         self.headers = headers
 
+        if self.headers["Content-Type"] == nil {
+            self.headers["Content-Type"] = "application/json"
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/swift5/APIs.mustache:37]: I don't think we should default the content-type to `application/json` as we've reports before that will not work in certain use cases (in other clients)

Comment (4brunu) [reply modules/openapi-generator/src/main/resources/swift5/APIs.mustache:37]: I copied that line from the kotlin client generator.
I will remove it.

**Comment by 4brunu** [comment]
CI is failing, but I think it's not related to this PR.

```
$ echo "$DOCKER_HUB_PASSWORD" | docker login --username=$DOCKER_HUB_USERNAME --password-stdin;
Must provide --username with --password-stdin
The command "echo "$DOCKER_HUB_PASSWORD" | docker login --username=$DOCKER_HUB_USERNAME --password-stdin;" failed and exited with 1 during .

Your build has been stopped.
```

**Comment by wing328** [comment]
Agreed it's not related.

**Pull request merged**